Airwallex/Core/Sources/Internal/Extensions/NSData+Base64.h

```diff
@@ -0,0 +1,19 @@
+//
+//  NSData+Base64.h
+//  Core
+//
+//  Created by Hector.Huang on 2023/3/17.
+//  Copyright © 2023 Airwallex. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSData (Base64)
+
+/**
+ In most base64 decoding implementations like Java, the padding-character is not needed, but ``NSData.initWithBase64EncodedString`` returns nil if it's missing.
+ This method will append "=" to base64 string whenever necessary and then decode to `NSData`.
+ */
++ (instancetype)initWithBase64NoPaddingString:(NSString *)base64String;
```

Comment:
Has this been used anywhere?

Reply:
Yep in [here](https://github.com/airwallex/airwallex-payment-ios/blob/43d83158e037ae8705c193fc7b7395f294a0b2ff/Airwallex/Core/Sources/AWXAnalyticsLogger.m#L116)